packages/logs/src/boot/logsPublicApi.ts

```diff
@@ -157,16 +118,16 @@ export function makeLogsPublicApi(startLogsImpl: StartLogs) {
 
     clearUser: monitor(() => userContextManager.clearContext()),
   })
+}
 
-  function overrideInitConfigurationForBridge<C extends InitConfiguration>(initConfiguration: C): C {
-    return assign({}, initConfiguration, { clientToken: 'empty' })
-  }
-
-  function canInitLogs(initConfiguration: LogsInitConfiguration) {
-    if (isAlreadyInitialized) {
-      displayAlreadyInitializedError('DD_LOGS', initConfiguration)
-      return false
-    }
-    return true
-  }
+function createActualStrategy(initConfiguration: LogsInitConfiguration, startLogsResult: StartLogsResult): Strategy {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏what about using the same naming between RUM and logs? (actual VS postStart)